README.md

```diff
@@ -31,16 +31,44 @@ Clipboard Utilities
  - Windows - clip
  - Linux   - xsel
 
+Commands
+--------
+
+System copy adds vim commands for copying `:SystemCopy`,
+`:SystemCopyLine` as well as for pasting `:SystemPaste`.
```

Comment:
Users can map to the `<Plug>` mappings directly so I don't need to provide a command.